docs/docs/setup-vast/tune.md

```diff
@@ -0,0 +1,219 @@
+to 2.0% at a false positive rate of `0.001`.
+
+The `avg-partition-disk-size` is in turn a function of the input data
+distribution and `max-partition-size` config option. It has been measured at
+about 163 MiB for a typical suricata eve.log.
+
+Finally, the `avg-partition-mem-size` can be conservatively calculated as twice
+the size of the `avg-partition-disk-size`. Multiplied to the value of
+`max-resident-partitions + 1` from the configuration file we get the maximum
+occupancy of the partitions.
+
+#### Example
+
+Putting in the measured example values and a `disk-budget-high` setting of
+2000 GiB we can calculate the estimated memory consumption to
+
+```
+11 * 1 Gib + 11 * 0.326 Gib + 0.02 * 2000 Gib
+11 Gib + 3.6 Gib + 40 Gib
+54.6 Gib
+```
+
+As you can see, depending on the disk budget and the entropy in the data the
+sketches that make up the catalog can quickly become the largest contributor to
+the memory requirement. Increasing the `max-partition-size` from the default 1
+Mib to 8 Mib reduces the sketch factor to `0.016` for our sample data, if we
+reduce the `max-resident-partitions` to `2` for a fair comparison, we get the
+the following new composition:
+
+```
+11 * 1 Gib + 3 * 2.608 Gib + 0.016 * 2000 Gib
+11 Gib + 7.2 Gib + 32 Gib
+50.2 Gib
+```
+
+## Shutdown
+
+The `stop` command gracefully brings down a VAST server that has been started
+with the `start` command.
+
+It is also possible to send a signal `SIGINT(2)` to the `vast` process instead
+of using `vast stop`, but in only works on the same machine that runs the
+server process. We recommend using `vast stop`, as it also works over the wire.
+
+The `stop` command blocks until the server process has terminated, and returns
+a zero exit code upon success, making it suitable for use in launch system
+scripts.
+
+The configuration option `vast.shutdown-grace-period` sets the time to wait
+until component shutdown finishes cleanly before inducing a hard kill.
+
+:::note
+The server waits for ongoing import processes to terminate before shutting down
+itself. In case an import process is hanging, you can always terminate the
+hanging process manually to shutdown the server.
+:::
+
+## Logging
+
+The VAST server writes log files into a file named `server.log` in the database
+directory by default. Set the option `vast.log-file` to change the location of
+the log file.
+
+VAST client processes do not write logs by default. Set the option
+`vast.client-log-file` to enable logging. Note that relative paths are
+interpreted relative to the current working directory of the client process.
+
+Server log files rotate automatically after 10 MiB. The option
+`vast.disable-log-rotation` allows for disabling log rotation entirely, and the
+option `vast.log-rotation-threshold` sets the size limit when a log file should
+be rotated.
+
+VAST processes log messages in a dedicated thread, which buffers up to 1M
+messages by default. The option `vast.log-queue-size` controls this setting.
```

Comment:
This default only applies to the server process. For client processes, it is 100.